src/index.js

```diff
@@ -47,7 +47,7 @@ class Dotenv {
     }
 
     Object.keys(blueprint).map(key => {
-      const value = env[key]
+      const value = env[key] || vars[key]
```

Comment:
The way it is set up now you cannot. Looking at dotenv-safe it looks like as long as the variable is set to anything it passes the safe check.

So back to your question... perhaps we should allow you to override a system car locally even if it is an empty string.

I'll look into that this afternoon.